math/mathcore/src/SparseData.cxx

```diff
@@ -193,28 +192,39 @@ namespace ROOT {
 
       SparseData::SparseData(vector<double>& min, vector<double>& max)
       {
-         // Creates a SparseData convering the range defined by min
+         // Creates a SparseData covering the range defined by min
          // and max. For this it will create an empty Box for that
          // range.
          Box originalBox(min, max);
-         fList = new ProxyListBox();
+         fList = std::make_unique<ProxyListBox>();
          fList->PushBack(originalBox);
       }
 
       SparseData::SparseData(const unsigned int dim, double min[], double max[])
       {
-         // Creates a SparseData convering the range defined by min
+         // Creates a SparseData covering the range defined by min
          // and max. For this it will create an empty Box for that
          // range.
          vector<double> minv(min,min+dim);
          vector<double> maxv(max,max+dim);
          Box originalBox(minv, maxv);
-         fList = new ProxyListBox();
+         fList = std::make_unique<ProxyListBox>();
          fList->PushBack(originalBox);
       }
 
-      SparseData::~SparseData()
-      { delete fList; }
+      SparseData::SparseData(const SparseData & rhs) :
+         FitData(rhs)
+      {
+         fList = std::make_unique<ProxyListBox>(*rhs.fList);
+      }
+      SparseData & SparseData::operator=(const SparseData & rhs)
+      {
+         FitData::operator=( rhs );
+         fList.reset(new ProxyListBox(*rhs.fList));
```

Comment:
```suggestion
         fList = std::make_unique<ProxyListBox>(*rhs.fList);
```
I think like in the other changes you made, `make_unique` should be preferred to not have the explicit `new`.